src/external/TinyMT/tinymt32.c

```diff
@@ -64,13 +64,14 @@ void tinymt32_init(tinymt32_t * random, uint32_t seed) {
     random->status[1] = random->mat1;
     random->status[2] = random->mat2;
     random->status[3] = random->tmat;
-    for (int i = 1; i < MIN_LOOP; i++) {
+    int i;
```

Comment:
We should move this to the very top of the function to be fully C compatible. Otherwise, this fix is good with me.